packages/manager/src/features/Kubernetes/KubeCheckoutBar/NodePoolSummaryItem.tsx

```diff
@@ -55,6 +64,7 @@ export const NodePoolSummaryItem = React.memo((props: Props) => {
         </Box>
         <Box mb={1.5} mt={2}>
           <EnhancedNumberInput
+            max={enterprisePrice ? MAX_NODES_PER_POOL_ENTERPRISE_TIER : 100}
```

Comment:
```suggestion
            max={enterprisePrice ? MAX_NODES_PER_POOL_ENTERPRISE_TIER : MAX_NODES_PER_POOL_STANDARD_TIER}
```
Nit just to use the constant for both maxes